lib/index.js

```diff
@@ -355,52 +354,48 @@ Server.prototype.reload = function(mixed) {
 }
 
 Server.prototype.stop = function(cb) {
-  if (this.app.started) {
-    _.each(this.app.interfaces, function(intf) {
-      if (
-        intf !== null &&
-        typeof intf === 'object' &&
-        typeof intf.stop === 'function'
-      ) {
-        intf.stop()
-      }
-    })
-
-    debug('Closing server...')
+  return new Promise((resolve, reject) => {
```

Comment:
Nothing rejects the Promise even if something goes wrong in shutdown. A try catch would handle synchronous errors.